src/state/application/reducer.ts

```diff
@@ -93,4 +104,12 @@ export default createReducer(initialState, builder =>
     .addCase(setGasPrice, (state, { payload: gasPrice }) => {
       state.gasPrice = gasPrice as GasPrice
     })
+    .addCase(toggleLiveChart, (state, action) => {
```

Comment:
I think this should belong to user's state, not application's state. 
https://github.com/dynamic-amm/dmm-interface/blob/main/src/state/user/reducer.ts#L63..L75
And currently, we are persisting user setting, so if you move this to user's state, you do not need to set it manually. 
https://github.com/dynamic-amm/dmm-interface/blob/main/src/state/index.ts#L18

Reply:
This is the last thing I wanna change @XiaoYhun, These other things look good to me :D

Reply:
okay didn't recognize it, thanks